static/src/stylesheets/module/content/_gallery.scss

```diff
@@ -297,37 +361,67 @@
         }
     }
 
-    // overriding defaults
-    .meta__save-for-later {
-        position: relative;
-        left: 0;
-    }
-
     // TODO: Replace all media tones instead of overwriting the defaults
     &.tonal--tone-media {
 
         // Overwriting media tone header so that you can see the image instead
         .tonal__header {
-            background-color: $multimedia-support-5
+            background-color: $multimedia-support-5;
+
+            .content__headline {
+                font-weight: 200;
+                padding-top: 0;
+            }
         }
 
         .meta__numbers {
             color: $neutral-3;
         }
 
         .content__dateline,
-        .sharecount__heading {
+        .sharecount__heading,
+        .commentcount2__heading {
             color: $neutral-2;
         }
 
+        .byline,
         .byline a,
-        .sharecount__value {
+        .sharecount__value,
+        .commentcount2__value {
             color: $neutral-3;
         }
 
+        .inline-comment-16 {
+            fill: $neutral-2;
+        }
+
+        a.commentcount2 {
+            &:hover {
+                text-decoration: none;
+            }
+        }
+
         .save-for-later__button--save .inline-icon {
             background-color: $multimedia-support-5;
         }
+
+        .save-for-later__button--saved .inline-icon,
+        .save-for-later__button--save:hover .inline-icon {
+            fill: $multimedia-support-5;
+        }
+
+        .tonal__main--tone-media .byline,
+        .tonal__main--tone-media .content__dateline,
+        .tonal__main--tone-media .content__meta-container,
+        .tonal__main--tone-media .ad-slot--paid-for-badge--article,
+        .tonal__main--tone-media .submeta hr,
+        .tonal__main--tone-media .commentcount,
+        .tonal__main--tone-media .meta__numbers,
+        .tonal__main--tone-media .meta__social,
+        .tonal__main--tone-media .meta__number:not(u-h)+.meta__number,
+        .ad-slot--dark .ad-slot__label {
+            border-color: $neutral-1;
+        }
```

Comment:
Would like to investigate all the specificity going on here but as we will be replacing the default media styles shortly, I think this is ok.